addons/calendar/data/mail_template_data.xml

```diff
@@ -371,7 +371,7 @@
         </record>
 
         <record id="calendar_template_meeting_update" model="mail.template">
-            <field name="name">Calendar: Event Update</field>
+            <field name="name">Appointment updated</field>
```

Comment:
This is not specific to appointment... isn't it confusing, as appointment is a specific app ?

Reply:
I went too fast and already changed some names, I believe PDE is still working on it, you may eventually discuss that with him. Indeed it seems confusing if it's not only used for appointments.
FYI you can see a full list of the intended changes on a google spreadsheet linked on the task :)